contracts/src/OffChainMessageContracts/ValidatorSetSig.sol

```diff
@@ -0,0 +1,130 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.18;
+
+import {
+    WarpMessage,
+    IWarpMessenger
+} from "@avalabs/subnet-evm-contracts@1.2.0/contracts/interfaces/IWarpMessenger.sol";
+import {ReentrancyGuard} from "@openzeppelin/contracts@4.8.1/security/ReentrancyGuard.sol";
+
+/**
+ * @dev Message format for the WarpMessage payload to be forwarded to the target contract
+ *
+ * validatorSetSigAddress: Address of the ValidatorSetSig contract this message is intended for
```

Comment:
we try to use the [NatSpec](https://docs.soliditylang.org/en/latest/natspec-format.html) format whenever possible, so these struct variables we use `@param`

Reply:
Yeah I looked at NatSpec but looks like the `@param` is only supported for functions, events and errors (correctly since these aren't params really)  hence why I went with this for now. It looks like this issue is open though: https://github.com/ethereum/solidity/issues/12295 

Do we still want to use `@param` even though it's a misnomer?